### PR TITLE
chore: update docs footer

### PR DESCRIPTION
**Pull request by deekshas8**
## What Has Changed?

Closes: https://github.com/SAP/cloud-sdk-backlog/issues/869

Can only be merged after #1042 as it needs the support page links.

## Manual Checks?

- [x] Check spelling and grammar, e.g., using Grammarly.
- [x] Verify links still work, e.g., if `id` or the name of a file is changed.
- [x] Update the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js), e.g., if a new feature is added.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -100,57 +100,61 @@ module.exports = {
...
-              label: 'Stack Overflow',
-              href: 'https://stackoverflow.com/questions/tagged/sap-cloud-sdk'
+              label: 'Tutorials',
+              to: 'docs/js/tutorials/getting-started/introduction'
             },
             {
-              label: 'SAP Developers Portal',
-              href: 'https://developers.sap.com/topics/cloud-sdk.html'
+              label: 'npm',
+              href: 'https://www.npmjs.com/search?q=%40sap-cloud-sdk'
```
Comment (FrankEssenberger) [inline docusaurus.config.js:128]: [q] why there is a `www` in the link. It also works without and non of the other http reference have it?

**Review by FrankEssenberger — APPROVED** [review]
LGTM only a uqestion

File: docusaurus.config.js (hunk 1 above)
Comment (deekshas8) [reply docusaurus.config.js:128]: The npmjs url has `www`. The url https://npmjs.com/search?q=%40sap-cloud-sdk redirects to https://www.npmjs.com/search?q=%40sap-cloud-sdk

File: docusaurus.config.js [hunk 2]
```diff
@@ -100,57 +100,61 @@ module.exports = {
...
+              label: 'SAP Cloud SDK Community Page',
+              href: 'https://community.sap.com/topics/cloud-sdk'
+            },
+            {
+              label: 'SAP Cloud SDK on SAP Answers',
+              href: 'https://answers.sap.com/tags/73555000100800000895'
+            },
+            {
+              label: 'SAP Cloud SDK on SAP Help',
+              href: 'https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US?task=discover_task'
```
Comment (marikaner) [inline docusaurus.config.js:157]: [req] Not really related to your PR, but can we add this?
```suggestion
            {
              label: 'SAP Cloud SDK on SAP Help',
              href: 'https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US?task=discover_task'
            },
            {
              label: 'Feedback',
              href: 'https://sapinsights.eu.qualtrics.com/jfe/form/SV_0P69X6kJ0Pdxqbc'
```

**Pull request merged**